create_release.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+git fetch && git rebase origin/master
```

Comment:
maybe you want to ensure you are in master. an alternative is to change that to `git fetch && git reset --hard origin/master` in which case it won't matter.

Reply:
I don't it! I'm pushing for `release-1.0.2`, I think if I reset-hard I'll lose all the commits from this branch

Reply:
Ah, I see what you mean. I could alternatively checkout to master? 

```git fetch && git checkout/master && git reset --hard origin/master```?

Reply:
I thought you were doing it from master, which makes no sense. Leave as is.